xbmc/guilib/DirtyRegion.h

```diff
@@ -35,3 +35,10 @@ class CDirtyRegion : public CRect
 };
 
 typedef std::vector<CDirtyRegion> CDirtyRegionList;
+
+class CRenderingStats
+{
```

Comment:
Okay. What do you think about if we create a new container class where we put all the stuff (dirty time, dirty region list, render stats) in, something​ like `ProcessInfo` and use it as parameter in `Process`. This way we don't need to alter the method signature of `Process` again and again if we need to add something else, in the future Then we could rename the header file too.